SQL/0000-00-00-schema.sql

```diff
@@ -2041,3 +2041,24 @@ CREATE TABLE `candidate_consent_history` (
   `EntryStaff` varchar(255) DEFAULT NULL,
   CONSTRAINT `PK_candidate_consent_history` PRIMARY KEY (`CandidateConsentHistoryID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `visit` (
+  `VisitID` int(10) unsigned NOT NULL auto_increment,
+  `VisitName` varchar(100) NOT NULL,
+  CONSTRAINT `visit_PK` PRIMARY KEY (`VisitID`),
+  CONSTRAINT `visit_name_UK` UNIQUE KEY (`VisitName`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+CREATE TABLE `visit_project_subproject_rel` (
+ `VisitID` int(10) unsigned NOT NULL,
+ `ProjectID` int(2) NOT NULL,
+  `SubProjectID` int(10) unsigned NOT NULL,
```

Comment:
The spacing here is different than the other rows in the table. Also, `Subproject` has a lowercase P. (SAme with all the other references to it.)